release-notes-1.2.0.md

```diff
@@ -0,0 +1,766 @@
+# Overview of jBallerina 1.2.0
+
+JBallerina 1.2.0 is the first major Q1 release of 2020 and it includes a new set of features and significant improvements to the compiler, runtime, standard library and developer tooling. It is based on Stable Language Spec version 2020R1.
```

Comment:
Saying "first major Q1 release" is redundant since we are doing one per quarter anyway. Need to rephrase this in a better way

Reply:
Also, I think it's sufficient to say "It is based on the language specification version 2020R1"

Reply:
How about saying "JBallerina 1.2.0 is the first major release of 2020 and it includes ..."?

Reply:
@hasithaa

Reply:
Looks good to me.

Reply:
Fixed.